static/src/js/components/PublishPreview/__tests__/__mocks__/publishPreview.js

```diff
@@ -439,11 +444,12 @@ export const noTagsOrGranulesOrServicesCollection = {
   associatedDois: [{
     doi: 'TestDOI'
   }],
-  collectionCitations: [{
```

Comment:
Double check this because I think the collectionCitations might be a different component of the collection there is a field for that here https://github.com/nasa/cmr-graphql/blob/main/src/types/collection.graphql#L42 in which case I think we'd use both in this mock